doc/source/running.rst

```diff
@@ -43,10 +43,8 @@ log messages.
 
 .. tip::
    
-   Running as the mongod user and saving the log file in the same parent
-   directory as the mongod node's data directory would be the most intuitive and
-   convenient way. But if you want it can be another user, and the log file can
-   be at location that files can be written to, or piped to logging service.
+   Running as the mongod user would be the most intuitive and convenient way.
+   But if you want it can be another user.
```

Comment:
Put a couple of commas to make it more readable: But, if you want, it can be another user. 
Or, to make this fragment even more readable: But you can run `pb-agent` as another user, too.